js/src/data/actions.js

```diff
@@ -1247,3 +1247,66 @@ export function* upsertTour( tour, upsertingClientStoreFirst = false ) {
 		);
 	}
 }
+
+export function receiveAcceptedTerms( data ) {
+	return {
+		type: TYPES.RECEIVE_ACCEPTED_CUSTOMER_DATA_TERMS,
+		data,
+	};
+}
+
+/**
+ * Updates the enhanced ads conversion status.
+ *
+ * @param {string} status to enable/disable enhanced ads conversion. Possible values are pending, enabled and disabled.
+ * @return {boolean} String The updated status.
+ */
+export function* updateEnhancedAdsConversionStatus( status ) {
+	try {
+		const response = yield apiFetch( {
+			path: `${ API_NAMESPACE }/ads/enhanced-conversion-status`,
+			method: REQUEST_ACTIONS.POST,
+			data: {
+				status,
+			},
+		} );
+
+		return {
+			type: TYPES.RECEIVE_ALLOW_ENHANCED_CONVERSIONS,
+			data: response,
+		};
```

Comment:
What about replacing this with calling `receiveAllowEnhancedConversions`?

```suggestion
		return receiveAllowEnhancedConversions( response );
```

Reply:
Sure thing 😁